Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -77,6 +77,17 @@ private final class TokenStreamCreator: SyntaxVisitor {
     afterMap[tok, default: []].append(tokens)
   }
 
+  private func insertTokenBetween<Node: Collection>(
```

Comment:
Naming nit (to make it Swiftier): `insertToken(_ token: Token, betweenChildrenOf collectionNode: Node)`

Then call sites become something like `insertToken(.break, betweenChildrenOf: node)`